.github/workflows/release.yml

```diff
@@ -44,6 +44,14 @@ jobs:
       - name: Zip files for release
         run: |
```

Comment:
Strongly suggest adding `shell: bash` before the `run:` line. Reason being that if you don't, then the default shell for that runner is used - and that might be something else, which may not support your file exists expressions.

Reply:
Thanks for your advice